sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2DataFrameSuite.scala

```diff
@@ -498,43 +498,32 @@ class DataSourceV2DataFrameSuite
              |""".stripMargin)
 
         val alterExecCol1 = executeAndKeepPhysicalPlan[AlterTableExec] {
-          sql(s"ALTER TABLE $tableName ALTER COLUMN salary SET DEFAULT (123 + 56)")
-        }
-        checkDefaultValue(
-          alterExecCol1.changes.collect {
-            case u: UpdateColumnDefaultValue => u
-          }.head,
-          new DefaultValue(
-            "(123 + 56)",
-            new GeneralScalarExpression(
-              "+",
-              Array(LiteralValue(123, IntegerType), LiteralValue(56, IntegerType)))))
-
-        val alterExecCol2 = executeAndKeepPhysicalPlan[AlterTableExec] {
-          sql(s"ALTER TABLE $tableName ALTER COLUMN salary SET DEFAULT ('r' || 'l')")
-        }
-        checkDefaultValue(
-          alterExecCol2.changes.collect {
-            case u: UpdateColumnDefaultValue => u
-          }.head,
-          new DefaultValue(
-            "('r' || 'l')",
-            new GeneralScalarExpression(
-              "CONCAT",
-              Array(
-                LiteralValue(UTF8String.fromString("r"), StringType),
-                LiteralValue(UTF8String.fromString("l"), StringType)))))
-
-        val alterExecCol3 = executeAndKeepPhysicalPlan[AlterTableExec] {
-          sql(s"ALTER TABLE $tableName ALTER COLUMN salary SET DEFAULT CAST(0 AS BOOLEAN)")
+          sql(
```

Comment:
Some test cleanup, I didnt realize multiple ALTER COLUMN can be the same statement, so it was unnecessarily long